client/locales/en-US.json

```diff
@@ -11,6 +11,7 @@
   "artist": "Artist",
   "artists": "Artists",
   "audio": "Audio",
+  "backgroundInformation": "Background information",
```

Comment:
I'm not sure about the wording of this, it's true that in the Wikipedia screenshot posted it appears as a "Background information", but I'm not able to find any other articles about actors that have that wording. There's simply no "Background" in that information, it's just "usual" information you would expect from a person, if that makes sense.

I see "Details" or "Personal information" more fitting.